src/main/java/org/fogbowcloud/manager/core/models/orders/ComputeOrder.java

```diff
@@ -39,14 +42,15 @@ public ComputeOrder(
             int disk,
             String imageId,
             UserData userData,
-            String publicKey) {
+            String publicKey, List<String> networksId) {
```

Comment:
Since the parameters have a line break after each one, I think you should also add a line break after publickKey parameter